src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 40px;
+  padding-right: 50px;
+  align-items: center;
+  line-height: 14px;
+}
+
+img {
+  align-self: center;
+  display: block;
+  margin-left: 10px;
+}
```

Comment:
1. `align-self: center;` and `margin-left: 10px;` is redundant (if you do `padding: 0 50px`)
2. Add sizes to image